### PR TITLE
Update android_workflow.dart

### LINKED ISSUE #2157: Insufficient flutter doctor instructions
`flutter doctor` wants the Android SDK, but doesn't say where to put it or how to point to it. The Android Setup section on the web site at  https://flutter.io/setup/#android-setup says to install Android Studio.  Based on a conversation with abarth, it appears that installing just the SDK is not sufficient - you also need the JDK.

(Maybe it's also worth saying that ANDROID_HOME is not set in case the user decides to install everything manually because ANDROID_HOME won't be set in existing terminal windows.)

Here is the text displayed by `flutter doctor`:
```
[ ] Android toolchain - develop for Android devices (missing)
  [ ] Android SDK - enable development for Android devices (missing)
      Download at https://developer.android.com/sdk/
```

### Steps to Reproduce
Run `flutter doctor` on a system where the Android SDK is not properly set up.

### Flutter Version
```
Flutter from git@github.com:flutter/flutter.git (on master)
flutter revision: 4860e63f444cf3786a6b136430f3058a7b851041 (8 hours ago)
engine revision : a53faaf7983025f83a2059021a409c6e98beb58d
```

### PR DESCRIPTION
**Pull request by devoncarew**
fix https://github.com/flutter/flutter/issues/2157

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
lgtm

File: packages/flutter_tools/lib/src/android/android_workflow.dart [hunk 1]
```diff
@@ -28,7 +28,8 @@ class AndroidWorkflow extends Workflow {
     androidValidator.addValidator(new Validator(
       'Android SDK',
```
Comment (abarth) [inline packages/flutter_tools/lib/src/android/android_workflow.dart]: We should change this to say "Android Studio"

Comment (devoncarew) [inline packages/flutter_tools/lib/src/android/android_workflow.dart]: Perhaps in the download instructions? This check will pass if we can locate an Android SDK.

Comment (jimbeveridge) [inline packages/flutter_tools/lib/src/android/android_workflow.dart]: This says SDK, but the web page we direct them to says "Android Studio". This is confusing.

Comment (abarth) [inline packages/flutter_tools/lib/src/android/android_workflow.dart]: It's really this label that was confusing Jim.  He went to that page and tried to install the "Android SDK", which then presented him with more questions like "where should this be installed?" that he didn't know how to answer.  Installing "Android Studio" removes the questions from the equation.

The situation here is somewhat subtle.  We require only the SDK but the way we recommend getting the SDK is to install a larger thing that includes the SDK.

Comment (devoncarew) [inline packages/flutter_tools/lib/src/android/android_workflow.dart]: I updated it to read `Android Studio / Android SDK`. That way we're not checking for the sdk, but telling people that studio is installed.

**Pull request merged**